### PR TITLE
Ticket time issues fixed

### PR DESCRIPTION
**Pull request by KNaveenraj-ladybird**
Fixes -#8213

### CODE REVIEW

**Comment by Sabaric12** [comment]
@KNaveenraj-ladybird Get the code approval for this PR

**Review by ajaz-ul-haq — DISMISSED** [review]

File: app/Http/Controllers/Agent/helpdesk/TicketController.php [hunk 1]
```diff
@@ -437,7 +437,7 @@ public function reply(Ticket_Thread $thread, Request $request, Ticket_attachment
 
             return response()->json(compact('result'));
         }
-        $result = ['success' => 'Replyed successfully'];
+        $result = ['success' => 'Replied successfully'];
```
Comment (RafficMohammed) [inline app/Http/Controllers/Agent/helpdesk/TicketController.php:440]: Use trans() or Lang::get();

**Review by RafficMohammed — CHANGES_REQUESTED** [review]
Write a test cases for these changes

File: resources/views/themes/default1/agent/helpdesk/ticket/timeline.blade.php [hunk 2]
```diff
@@ -1824,20 +1824,18 @@ function formatBytes(bytes,decimals) {
...
-                    $("#alert21").show();
-
-                    $('#message-success2').html(json.result.success);
                 }, 1000);
 
-
-                $('html, body').animate({ scrollTop: $("#inboxactions").offset().top }, 500);
+                $("#alert21").show();
+                $('#message-success2').html(json.result.success);
+                //$('html, body').animate({ scrollTop: $("#inboxactions").offset().top }, 500);
```
Comment (DhanyaChengappa) [inline resources/views/themes/default1/agent/helpdesk/ticket/timeline.blade.php:1838]: if this line is not required, lets remove

**Review by DhanyaChengappa — CHANGES_REQUESTED** [review]

**Review by RafficMohammed — APPROVED** [review]

**Review by ajaz-ul-haq — APPROVED** [review]

**Comment by Sabaric12** [comment]
Tested this PR with the fresh installation
Issue 1:
While replying to the ticket created by the user replied agent or admin the success alert message is coming but it vanishes suddenly
image
@KNaveenraj-ladybird resolve the conflicts also
Round 1 Testing Summary:
Total time in days: 3 hours
Total issues: 1
Minor - 0 
Major - 1
Critical - 0
No of times PR was sent for Functionality correction: 1
PR breaks existing functionality: No

**Comment by Sabaric12** [comment]
Today's task:-
Tested this PR with the fresh installation
The mentioned issues are working fine and one separate issue has been created
https://github.com/ladybirdweb/faveo-helpdesk/issues/8235
Round 1 Testing Summary:
Total time in days: 3 hours
Total issues: 0
Minor - 0 
Major - 0
Critical - 0
No of times PR was sent for Functionality correction: 0
PR breaks existing functionality: No

**Review by Sabaric12 — APPROVED** [review]

**Review by AbishekChinnaraju — APPROVED** [review]

**Comment by KNaveenraj-ladybird** [comment]
Hello ladybirdweb/faveo-helpdeskThank you for contacting us. This is an automated response confirming the receipt of your ticket. Our team will get back to you as soon as possible. When replying, please make sure that the ticket ID is kept in the subject so that we can track your replies.Ticket ID: AAAA-0000-0002  You can check the status of or update this ticket online at: https://localhost/faveo-helpdesk/public/check_ticket/eyJpdiI6InRTQmtpL1hBb0QrQTA4NDhGL1Q3Z0E9PSIsInZhbHVlIjoiM1dwUDBHZkFzNld5dXI5aSt6a2htUT09IiwibWFjIjoiOGZjNjBiOTdlY2M0ZGE0Y2ZkZjQxMjk1ODFjMDY3MTcyNWZkY2RjY2Q1MjFiMDYxZGY5NGVkYTBiYWIwNWY2ZiIsInRhZyI6IiJ9

**Comment by KNaveenraj-ladybird** [comment]
---Reply above this line--- Hello ***@***.***,A reply been made to ticket AAAA-0000-0002FromName: NaveenrajE-mail: ***@***.*** Hello Naveenraj,New ticket AAAA-0000-0002created FromName: ladybirdweb/faveo-helpdesk   E-mail: ***@***.*** 
Merged #8219 into development.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/ladybirdweb/faveo-helpdesk/pull/8219#event-10776252602",
"url": "https://github.com/ladybirdweb/faveo-helpdesk/pull/8219#event-10776252602",
"name": "View Pull Request"
},
"description": "View this Pull Request on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]Kind Regards, Support Center
Kind Regards, Support Center

**Pull request merged**